<!-- 
This launch file includes the necessary nodes and parameters to run the skidpad detection algorithm using LIDAR data. 

The following nodes are included:
- velodyne_pointcloud/VLP-32C_points.launch: Launches the Velodyne LIDAR driver node to publish point cloud data.
- lidar_cluster/lidar_cluster.launch: Launches the node to cluster the LIDAR point cloud data.
- car_position/car_position.launch: Launches the node to publish the position of the car.
- skidpad_detection_node: Launches the node to detect skidpad using LIDAR data and publishes the results.
- rviz: Launches the RViz visualization tool with the configuration file to display the clustered LIDAR data.

The skidpad_detection.yaml file is loaded as a rosparam for the skidpad_detection_node.

-->
<launch>
    <include file="$(find velodyne_pointcloud)/launch/VLP-32C_points.launch"/>
    <include file="$(find lidar_cluster)/launch/lidar_cluster.launch"/>
    <include file="$(find car_position)/launch/car_position.launch"/>
    <node pkg="skidpad_detection" type="skidpad_detection_node" name="skidpad_detection_node" output="screen">
        <rosparam command="load" file="$(find skidpad_detection)/config/skidpad_detection.yaml" />
    </node>
    <node pkg="rviz" type="rviz" name="my_rviz" args="-d $(find lidar_cluster)/config/lidar_cluster.rviz"/>

    <node name="filter_bagplay" pkg="rosbag" type="play" output="screen" args="/home/zhangjszs/filter.bag">
  
  </node>
  
</launch>
<!--<launch>
    <include file="$(find velodyne_pointcloud)/launch/VLP-32C_points.launch"/>
    <include file="$(find lidar_cluster)/launch/lidar_cluster.launch"/>
    <include file="$(find cone_position)/launch/cone.launch"/>  
    <node pkg="cone_position" type="cone_position" name="cone_position"  output="screen"/>   
    <node pkg="car_position" type="car_position" name="car_position"  output="screen"/>
    <node name="urinay" pkg="urinay" type="urinay_exec" output="screen" ns="/AS/P">
        <rosparam command="load" file="$(find urinay)/config/urinay.yml" />
    </node>
    <node pkg="skidpad_detection" type="skidpad_detection_node" name="skidpad_detection_node" output="screen">
        <rosparam command="load" file="$(find skidpad_detection)/config/skidpad_detection.yaml" />
    </node>
    <node pkg="rviz" type="rviz" name="my_rviz" args="-d $(find lidar_cluster)/config/lidar_cluster.rviz"/>
</launch>-->



